.ci/Project.toml

```
@@ -8,6 +8,6 @@ TimeZones = "f269a46b-ccf7-5d73-abea-4c690281aa53"
 [compat]
 GitHub = "5"
 HTTP = "0.9, 1"
-RegistryCI = "7"
+RegistryCI = "7,8"
```

Actually, probably should only allow v8 (need to wait for registration though)
```suggestion
RegistryCI = "8"
```